28-04/task_2/cleanfile.rb

```diff
@@ -0,0 +1,14 @@
+class Cleanfile
+  attr_accessor :file_name
+
+  def initialize
+    @file_name = "palindrom.txt"
+  end
+
+  def open
+    File.open(@file_name).each do |line|
+      palindrome = line.strip.downcase.delete("\s.,")
```

Comment:
I think that parameter for delete should look like this `" .,"`. You can test that it's working fine by adding extra spaces between words in `palindrom.txt`